ui/src/app/components/help-sidebar.tsx

```diff
@@ -124,15 +125,15 @@ const iconStyles = {
 };
 
 interface Props {
-  location: string;
+  helpContent: string;
+  hideSidebar: number;
```

Comment:
This feels like it should semantically be a number. It looks like it gets incremented on navigate? Could we instead set it to true on every navigate? That feels more like what we are looking to do here.

Reply:
Renamed and changed to boolean along with reworking state/props based on your comment below